R/class-plotdata-bar.R

```diff
@@ -76,6 +78,7 @@ validateBarPD <- function(.bar, verbose) {
 #' @param variables veupathUtils VariableMetadataList
 #' @param value String indicating how to calculate y-values ('identity', 'count', 'proportion')
 #' @param barmode String indicating if bars should be grouped or stacked ('group', 'stack')
+#' @param overlayValues character vector providing overlay values of interest
```

Comment:
Consider adding that these values must exist in the overlay variable or else x, y , z bad things happen

Reply:
so i didnt bc its entirely possible we could be passed valid overlay values that we wont actually see in the subset..